Rakefile

```diff
@@ -1,13 +1,12 @@
 require 'rubygems'
 require 'bundler/setup'
-require 'rakeup'
 require 'resque/tasks'
 require './environment.rb'
 
-RakeUp::ServerTask.new do |t|
-  t.port = 1791
-  t.pid_file = 'tmp/server.pid'
-  t.server = :puma
+require 'rake/testtask'
+Rake::TestTask.new do |t|
+  ENV['APP_ENV'] ||= 'test'
+  t.pattern = 'test/**/*_test.rb'
 end
```

Comment:
This is not needed, we do it at line 41-47
https://github.com/Mapotempo/optimizer-api/pull/45/files#diff-ee98e028c59b193d58fde56ab4daf54d43c486ae674e63d50ddf300b07943e0fR41

Reply:
Well seen, I had a conflict with this file I did not notice it was duplicate.